go.mod

```diff
@@ -1,39 +1,48 @@
 module github.com/crate/cratedb-prometheus-adapter
 
-go 1.16
+go 1.18
+
+require (
+	github.com/go-kit/kit v0.7.1-0.20180619214315-e2d71a06a40a
+	github.com/golang/protobuf v1.5.2
+	github.com/golang/snappy v0.0.0-20180518054509-2e65f85255db
+	github.com/jackc/pgx v3.1.1-0.20180608201956-39bbc98d99d7+incompatible
+	github.com/prometheus/client_golang v0.9.0-pre1.0.20180623155954-77e8f2ddcfed
+	github.com/prometheus/common v0.0.0-20180518154759-7600349dcfe1
+	github.com/prometheus/prometheus v2.3.2-0.20180622142104-2bd510a63e48+incompatible
+	github.com/stretchr/testify v1.5.1
+	gopkg.in/yaml.v2 v2.4.0
+)
 
 require (
 	github.com/alecthomas/template v0.0.0-20150530000104-b867cc6ab45c // indirect
 	github.com/alecthomas/units v0.0.0-20150109002421-6b4e7dc5e314 // indirect
 	github.com/beorn7/perks v0.0.0-20180321164747-3a771d992973 // indirect
 	github.com/cockroachdb/apd v1.1.0 // indirect
 	github.com/davecgh/go-spew v1.1.1 // indirect
-	github.com/go-kit/kit v0.7.1-0.20180619214315-e2d71a06a40a
 	github.com/go-logfmt/logfmt v0.3.0 // indirect
 	github.com/go-stack/stack v1.7.0 // indirect
 	github.com/gogo/protobuf v1.3.2 // indirect
-	github.com/golang/protobuf v1.5.2
-	github.com/golang/snappy v0.0.0-20180518054509-2e65f85255db
 	github.com/grpc-ecosystem/grpc-gateway v1.16.0 // indirect
 	github.com/jackc/fake v0.0.0-20150926172116-812a484cc733 // indirect
-	github.com/jackc/pgx v3.1.1-0.20180608201956-39bbc98d99d7+incompatible
```

Comment:
Running `go mod tiny` introduced this second `require` block. It will be fine I guess.